docs/components/sidecar.md

```diff
@@ -228,6 +228,7 @@ Flags:
                                  configuration. See format details:
                                  https://thanos.io/tip/thanos/tracing.md/#configuration
       --tsdb.path="./data"       Data directory of TSDB.
+      --upload-jitter=0s         Maximum random delay before uploading blocks.
```

Comment:
Need to fix docs

Reply:
Also it looks like this flag is never consumed

Reply:
Also added nowhere

Reply:
yess will remove it